src/main/java/com/trello/BoardPage.java

```diff
@@ -112,26 +123,26 @@ public void closeBoard() {
         closeBoardBtn.click();
         waitUtils.waitElementToBeClickableShort(confirmationCloseBoardBtn);
         confirmationCloseBoardBtn.click();
-        waitUtils.waitInvisibilityOfElementShort(confirmationCloseBoardBtn);
     }
 
     public void deleteBoardPermanently() {
         closeBoard();
-        waitUtils.waitVisibilityOfElementShort(deleteBoard);
+        waitUtils.waitVisibilityOfElementLong(deleteBoard);
         deleteBoard.click();
         waitUtils.waitVisibilityOfElementShort(confirmationDeleteBoardBtn);
         confirmationDeleteBoardBtn.click();
-        waitUtils.waitInvisibilityOfElementShort(confirmationDeleteBoardBtn);
+
+        waitUtils.waitVisibilityOfElementLong(textAfterDeleteBoard);
     }
 
     public String getSearchFieldAttribute(String attributeName) {
         waitUtils.waitVisibilityOfElementShort(searchField);
         return searchField.getAttribute(attributeName);
     }
 
-    public String getBoardTitleAttribute(String attributeName) {
+    public String getBoardTitleAttribute() {
```

Comment:
Current method behavior does not match title

Reply:
Done